File: backend/components/metiers/administration_publique/utils/i18n/core/i18n.test.py
"""
i18n.test.py – Tests unitaires Python pour i18n Threed
"""
from .i18n import i18n

def test_i18n_fr():
    assert '[FR]' in i18n('Test', 'fr')

def test_i18n_en():
    assert '[EN]' in i18n('Test', 'en')

def test_i18n_fallback():
    assert i18n('Test', 'es') == 'Test'

File: backend/components/metiers/administration_publique/utils/i18n/core/i18n.test.js
// i18n.test.js – Tests unitaires JS pour i18n Threed
const i18n = require('./i18n');
describe('i18n Threed', () => {
  it('doit traduire en français', () => {
    expect(i18n.i18n('Test', 'fr')).toMatch('[FR]');
  });
  it('doit traduire en anglais', () => {
    expect(i18n.i18n('Test', 'en')).toMatch('[EN]');
  });
  it('doit fallback sur la valeur brute', () => {
    expect(i18n.i18n('Test', 'es')).toBe('Test');
  });
});
